README.adoc

```diff
@@ -460,26 +460,26 @@ external_services:
     var_workload: VALUE
 ----
 
-|`JAEGER_SERVICE_NAMESPACE`
-|The Kubernetes namespace that holds the Jaeger service. (default is `istio-system`)
+|`JAEGER_URL`
+|The URL to the Jaeger service. When not set, Kiali throw an error when the user try request to Jaeger (/api/jaeger).
 [source,yaml]
 ----
 external_services:
   jaeger:
-    service_namespace: VALUE
+    url: VALUE
 ----
 
-|`JAEGER_SERVICE`
-|The Kubernetes service name for Jaeger. (default is `jaeger-query`)
+|`JAEGER_SERVICE_NAMESPACE`
+|The Kubernetes namespace that holds the Jaeger service. (default is `istio-system`)
 [source,yaml]
 ----
 external_services:
   jaeger:
-    service: VALUE
+    service_namespace: VALUE
 ----
 
-|`JAEGER_SERVICE_PORT`
-|The Kubernetes service port for Jaeger. (default is `16686`)
+|`JAEGER_SERVICE`
```

Comment:
Same here, I don't think we need the JAEGER_SERVICE now.
Perhaps the info that "jaeger-query" is the service to expose should go into the JAEGER_URL variable.